src/panel/direction/DirectionPanel.jsx

```diff
@@ -310,13 +313,8 @@ export default class DirectionPanel extends React.Component {
         : <Panel
           className="direction-panel"
           onClose={this.onClose}
-          renderHeader={(
-            <div>
-              {title}
-            </div>
-          )}
+          renderHeader={form}
```

Comment:
Isn't it a bit dangerous to use the header for that?

Reply:
I would just put that in the children with the others, define no renderHeader at all, and set a `z-index: 1` to the Panel CloseButton.

Reply:
what do you mean by "dangerous" ? This is needed to separate the real, **scrollable** panel content, and its header which is really just a children.